qrisk/stats.py

```diff
@@ -15,13 +15,10 @@
 
 from __future__ import division
 
-from collections import OrderedDict
-from functools import partial
-
 import pandas as pd
 import numpy as np
 import scipy as sp
-import scipy.stats
+import scipy.stats  # noqa
```

Comment:
`scipy` does not import `stats` by default so it's required. flake8 is just not smart enough to track `scipy.stats` to `sp.stats`.